app/(gcforms)/[locale]/(form administration)/form-builder/[id]/responses/[statusFilter]/actions.ts

```diff
@@ -240,6 +261,36 @@ export const getSubmissionsByFormat = async ({
             } as Answer;
           }
 
+          // Handle "Split" AddressComplete in a similiar manner to dynamic fields.
+          if (
+            question?.type === FormElementTypes.addressComplete &&
+            question.properties.addressComponents?.splitAddress === true
+          ) {
+            const addressObject = JSON.parse(answer as string) as AddressElements;
+
+            const questionComponents = question.properties.addressComponents as AddressComponents;
+            if (questionComponents.canadianOnly) {
+              addressObject.country = "CAN";
+            }
+
+            const reviewElements = getAddressAsAnswerElements(
+              question,
+              addressObject,
+              extraTranslations
+            );
+
+            const addressElements = [reviewElements];
+
+            return {
+              questionId: question.id,
+              type: FormElementTypes.address,
+              questionEn: question?.properties.titleEn,
+              questionFr: question?.properties.titleFr,
+              answer: addressElements,
+            } as Answer;
+          }
```

Comment:
Still think the better place to handle this is in getAnswerAsString. You're really just returning one of two string representations of the stored data.